projects/storefrontapp-e2e-cypress/cypress/sample-data/checkout-flow.ts

```diff
@@ -32,11 +32,11 @@ export const product = {
 };
 
 export const cart = {
+  estimatedShipping: '$11.99',
   total: '$2,623.08',
```

Comment:
The value here isn't correct, when running the e2e tests, the cart has a value of $2,153.77.

Reply:
Which e2e test specifically? Checkout-flow e2e tests were passing except inconsistently checkout-as-persistent-user due to stability issues

Reply:
Works now